tests/unit/colorpickersplitbutton/tests.html

```diff
@@ -68,107 +68,68 @@
 	        
-	        equal(getDefaultChangedColor, color, "defaultColor value was not changed");
+	        equal(getDefaultChangedColor, color, "default color value wasn't changed");
 	    });
 
-	    test("_setOption", function () {
-	    	var color = "#afafaf";
-	    	this.actualColorPickerSplitButton.options.defaultColor = color;
-	    	var getDefaultColor = this.actualColorPickerSplitButton.options.defaultColor;
-
-	    	equal(getDefaultColor, color, "defaultColor value was not changed");
-	    })
-
-	    test("_setButtonColorIndicator", function () {
-	    	var color = "#ffffff";
-	    	this.actualColorPickerSplitButton._setButtonColorIndicator(color);
-
-	    	var getRgbColor = hexToRgb(color);
-
-	    	var getBtnColorIndicatorColor = this.actualColorPickerSplitButton.element.children(":first").children(":first").css("background-color");
-
-	    	this.actualColorPickerSplitButton._onItemClick();
-
-	    	equal(getBtnColorIndicatorColor, getRgbColor, "button color indicator was not set");
-	    })
-
-	    test("_onItemClick", function () {
-	    	equal(this.actualColorPickerSplitButton._onItemClick(), false, "when invoke _onItemClick isn't return false");
-	    })
-
 	    test("_onDefaultButtonClick", function () {
 	    	// Check before click if input element exist
-	    	var clickInputValue = this.actualColorPickerSplitButton.element.find("input").val();
-	    	var getDefaultButton = this.actualColorPickerSplitButton.element.children(":first").children(":first");
+	    	var clickInputValue = this.actualColorPickerSplitButton.igColorPickerSplitButton().find("input").val();
+	    	var getDefaultButton = this.actualColorPickerSplitButton.igColorPickerSplitButton().children(":first").children(":first");
 	    	
 	    	equal(clickInputValue, undefined, "default button was clicked");
 	    	
-	    	getDefaultButton.trigger("click");
+	    	getDefaultButton.click();
 	    	// Check after click the input element value
-	    	clickInputValue = this.actualColorPickerSplitButton.element.find("input").val();
+	    	clickInputValue = this.actualColorPickerSplitButton.igColorPickerSplitButton().find("input").val();
 
 	    	equal(clickInputValue, "clicked", "default button was not clicked");
-	    })
-
-	    test("_preventCollapsing", function () {
-	    	var isPreventCollapsing = this.actualColorPickerSplitButton._preventCollapsing();
-	    	equal(isPreventCollapsing, false, "when invoke _preventCollapsing method isn't return false");
-	    })
+	    });
 
 	    test("_setOptinWithEqualKeyValues", function () {
 	    	var color = "#afafaf";
 
-	    	notEqual(color, this.actualColorPickerSplitButton.options.defaultColor, "defaultColor option hasn't different value from given color value");
+	    	notEqual(color, this.actualColorPickerSplitButton.igColorPickerSplitButton("option", "defaultColor"), "defaultColor option hasn't different value from given color value");
```

Comment:
The assert parameters in QUnit are `assert(actual, expected, error message)` They are switched up in these tests. Please fix that.